src/main/java/com/amalitech/gpuconfigurator/model/Contact.java

```diff
@@ -0,0 +1,42 @@
+package com.amalitech.gpuconfigurator.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Data
```

Comment:
Don't use @Data annotation on Entities
You can use @Getter, @Setter, and manually create you Equals and Hashcode
The lombok @EqualsAndHashCode has performance issues